app/components/NavGuest.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
watch linting - i think we're doing 2 spaces not tabs

Reply:
Merge the Travis PR first, add linting to CI. Shouldn't have to catch this manually.

Reply:
I'm working on Travis CI now. I'll try to get it up b4 next pr.